python/ray/rllib/models/action_dist.py

```diff
@@ -4,9 +4,15 @@
 
 import tensorflow as tf
 import numpy as np
+import distutils.version
+
 from ray.rllib.utils.reshaper import Reshaper
 
 
+use_tf150_api = (distutils.version.LooseVersion(tf.VERSION) >=
```

Comment:
You might have to run scripts/yapf.sh to fix indentation

Reply:
applied yapf. sorry that I hv no idea about the following failure https://amplab.cs.berkeley.edu/jenkins//job/Ray-PRB/7303/